homeassistant/components/solaredge_local/sensor.py

```diff
@@ -148,8 +148,7 @@ def update(self):
         """Update the data from the SolarEdge Monitoring API."""
         try:
             response = self.api.get_status()
-            _LOGGER.debug("response from SolarEdge: %s", response)
-
+            _LOGGER.debug("response from SolarEdge: %s", response)            
             self.data["energyTotal"] = response.energy.total
```

Comment:
I was thinking about the access of these attributes. Should we move that out of this try except to a new try except where we just catch `AttributeError`? I'm guessing only `self.api.get_status()` does I/O.

Reply:
Gotcha, check the change if this is OK